android/src/main/java/cloud/pace/plugins/cloudsdk/CloudSDK.kt

```diff
@@ -39,7 +53,7 @@ class CloudSDK : Plugin(), AppCallback {
         val callEnvironment = call.getString(ENVIRONMENT) ?: PRODUCTION
 
         val authenticationMode = try {
-            AuthenticationMode.valueOf(callAuthenticationMode)
+            AuthenticationMode.valueOf(callAuthenticationMode.toUpperCase(Locale.getDefault()))
```

Comment:
I think ` uppercase()` is the way to go here.

Reply:
Actually I think we should get rid of the throwing enum instantiation completely.

Instead a simple `callAuthenticationMode.equals("native", true)` and then returning the corresponding enum seems cleaner.

Reply:
I'll merge this for now and you can do the refactoring in a follow-up.